.lando.yml

```diff
@@ -0,0 +1,15 @@
+name: starshot
+recipe: drupal10
+config:
+  webroot: web
+tooling:
+  drupal:install:
+    service: appserver
+    cmd: |
+      composer install
+      composer drupal:install
+      echo ""
+      echo "🚀 Board the spaceship!"
```

Comment:
This line is cute but I think we can probably remove it. :)

Reply:
:baby_chick: thank you... I agree BUT... I think emojis makes the experience much better. 

Can we keep it - please ?

Reply:
A poll we've done in Drupal Slack couple of months ago around using of Emojis in CLI output

<img width="330" alt="Search_-_Drupal_-_Slack" src="https://github.com/phenaproxima/starshot-prototype/assets/378794/29db2117-9537-42d0-8939-9069b2c2a505">

Reply:
So there is a majority of users feeling positive about emojis in CLI :) Thank you for this info @AlexSkrypnyk

Reply:
No objection to emoji. But let's make it a little briefer. I would suggest:

```
"🚀 Welcome to Starshot! Click on the link below to log in.👇"
```